<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFieldCloudLoginDialog</class>
 <widget class="QDialog" name="QFieldCloudLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFieldCloud Login Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="loginFormGroup" native="true">
     <layout class="QFormLayout" name="formLayout_3">
      <item row="1" column="1">
       <widget class="QLineEdit" name="usernameLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsPasswordLineEdit" name="passwordLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="loginFeedbackLabel">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="loginFormButtonsLayout" stretch="0,0,0,0">
        <item>
         <widget class="QCheckBox" name="rememberMeCheckBox">
          <property name="text">
           <string>Remember Me</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="loginFormButtonsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="serverUrlLabel">
        <property name="text">
         <string>Server URL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="serverUrlCmb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgspasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
